packages/nextjs/src/config/loaders/proxyLoader.ts

```diff
@@ -16,7 +17,7 @@ type LoaderOptions = {
  */
 export default async function proxyLoader(this: LoaderThis<LoaderOptions>, userCode: string): Promise<string> {
   // We know one or the other will be defined, depending on the version of webpack being used
-  const { pagesDir } = 'getOptions' in this ? this.getOptions() : this.query;
+  const { pagesDir, pageExtensionRegex } = 'getOptions' in this ? this.getOptions() : this.query;
```

Comment:
m: we don't have to set a default here?

Reply:
I mean it is always passed in very explicitly (see `webpack.ts`). Would it make sense to have another default here in your opinion?

Reply:
Ahhh ok my bad didn't see how the types were linked up - with `LoaderOptions` it makes sense now.